datafusion/src/sql/planner.rs

```diff
@@ -194,7 +194,7 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
             {
                 Ok(LogicalPlan::DropTable(DropTable {
                     name: names.get(0).unwrap().to_string(),
-                    if_exist: *if_exists,
+                    if_exist: if_exists,
```

Comment:
after you solve conflicts with https://github.com/apache/arrow-datafusion/pull/1951#discussion_r821440524
```suggestion
                    if_exists,
```